lib/uniris/beacon_chain/subset.ex

```diff
@@ -84,9 +84,9 @@ defmodule Uniris.BeaconChain.Subset do
       ) do
     listNodes =
       Enum.filter(P2P.list_nodes(), fn x -> :binary.part(x.first_public_key, 0, 1) == subset end)
-      Logger.info("List Node : ", listNodes)
 
-    _p2p_view_available = Enum.map(listNodes, fn x -> GenICMP.ping(x) end)
+    _p2p_view_available = Enum.map(listNodes, fn x -> GenICMP.ping(x.ip) end)
```

Comment:
I think you should use Task.async_stream to make the ping requests concurrently.
Otherwise it will do it sequentially, one after another.